backend/_handler.js

```diff
@@ -4,106 +4,110 @@ import * as db from '@reshuffle/db';
 const swaggerUi = require('swagger-ui-express');
 const swaggerDocument = require('./swagger.json');
 
-
 const app = express();
 
 // API endpoints
 const allPetsQuery = db.Q.filter(db.Q.key.startsWith('pets/'));
 
 //GET /collection - gets all entries in the collection, query string is the filter attributes like /collection?param1=foo
 app.get('/api/pets', async (req, res) => {
-    try {
-        const result = await db.find(allPetsQuery);
-        res.setHeader('Content-Type', 'application/json');
-        res.end(JSON.stringify(result));
-    } catch (e) {
-        res.sendStatus(500);
-        console.error(e);
-    }
+  try {
```

Comment:
All the functions are using `try {} catch { sendStatus 500 console.error }`, can it be extracted to a function?